homeassistant/components/reolink/number.py

```diff
@@ -468,12 +498,20 @@ async def async_setup_entry(
     """Set up a Reolink number entities."""
     reolink_data: ReolinkData = hass.data[DOMAIN][config_entry.entry_id]
 
-    async_add_entities(
+    entities: list[ReolinkNumberEntity | ReolinkChimeNumberEntity] = [
         ReolinkNumberEntity(reolink_data, channel, entity_description)
         for entity_description in NUMBER_ENTITIES
         for channel in reolink_data.host.api.channels
         if entity_description.supported(reolink_data.host.api, channel)
+    ]
+    entities.extend(
+        [
```

Comment:
```suggestion
```
You can remove the list creation and use a generator expression

Reply:
Thanks for the suggestion, I removed the list for all platforms of the reolink integration.